test/library/packages/Sort/performance/performance.chpl

```diff
@@ -47,7 +47,7 @@ proc stride(N) {
 proc gatherTimings(const ref A) {
   var t = new Timer();
   print('Time taken to sort ', 2**M, ' bytes (', A.size, ' ', T:string, 's)');
-  if sorts.find('q')
+  if sorts.find('q') != 0
```

Comment:
If you make `_cond_test` overloads for byteIndex and codePoint index then you won't need to add the `!= 0` here. I think that would be an improvement, at least for this pattern.